spec/cypress/support/step_definitions/common-navigation.js

```diff
@@ -1,8 +1,16 @@
-import { Given, When } from "cypress-cucumber-preprocessor/steps";
+import { Given, When, Then } from "cypress-cucumber-preprocessor/steps";
+
+Given("I am logged in as a {string}", (user) => cy.login(user));
+Given("I am logged in as an {string}", (user) => cy.login(user));
+
+Given("scenario {string} has been ran", (scenario) => cy.appScenario(scenario));
 
 const pagePaths = {
   cookie: "/cookies",
   start: "/",
+  "admin admin": "/admin/administrators",
+  "admin admin creation": "/admin/administrators/new",
+  "admin admin confirm creation": "/admin/administrators/new/confirm",
```

Comment:
roger roger...

`admin index`, `admin creation`, `admin confirm creation`?

Reply:
yea i hate it too. the other alternative i was thinking about is "admin management" which assumes that admins are the only people who can create admins - which isn't an unfair assumption. it's super long though!